plugins/Feedback/templates/index.twig

```diff
@@ -56,50 +53,50 @@
              content-title="{{ headline }}"
              feature="{{ 'General_Help'|translate }}">
             <p>{{ 'General_MatomoIsACollaborativeProjectYouCanContributeAndDonateNextRelease'|translate(
-                    "<a target='_blank' rel='noreferrer noopener' href='https://matomo.org'>",
+                    externallink("https://matomo.org"),
                     "</a>",
-                    "<a target='_blank' rel='noreferrer noopener' href='https://matomo.org/get-involved/'>",
+                    externallink("https://matomo.org/get-involved/"),
                     "</a>",
                     "<a href='" ~ linkTo({'module':'CoreAdminHome', 'action': 'home'})~"'>",
                     "</a>",
-                    "<a target='_blank' rel='noreferrer noopener' href='https://matomo.org/team/'>",
+                    externallink("https://matomo.org/team/"),
```

Comment:
this is a twig template, don't think using the javascript method would work correctly here.

Reply:
There are both twig template filters `externallink` and `externalrawlink`, and also TypeScript functions `externalLink` and `externalRawLink` added in this PR (twig stuff was in the first commit). So here it's the twig filters that are being called inside `{{` braces `}}`